public/sass/_variables.light.generated.scss

```diff
@@ -183,6 +183,8 @@ $btn-drag-image: '../img/grab_light.svg';
 
 $navbar-btn-gicon-brightness: brightness(1.5);
 
+$btn-active-box-shadow: 0px 0px 4px rgba(234, 161, 51, 0.6);
```

Comment:
`$btn-active-box-shadow ` created for both theme as this box shadow is used in `explore-active-button-glow` that us used for all active-buttons glows (timesync, timepicker, live-tailing button)